uvtools/dspec.py

```diff
@@ -849,7 +849,10 @@ def gen_window(window, N, alpha=0.5, edgecut_low=0, edgecut_hi=0, **kwargs):
         edgecut_hi : int, number of bins to consider as zero-padded at the high-side
             of the array, such that the window smoothly connects to zero.
         alpha : if window is 'tukey', this is its alpha parameter.
+        normalize : str, optional
+            set to 'rms' to divide by rms and 'mean' to divide by mean.
     """
+    assert normalization in ['none', 'mean', 'rms'], "normalization must be one of ['none', 'rms', 'mean']"
```

Comment:
I'm also a big proponent of foregoing `assert` statements when sanitizing user input (because you can run python in such a way that they are ignored) and instead explicitly doing something like:
```
if normalization not in ["mean", "rms"]:
    raise AssertionError("My message goes here")
```
This makes sure potentially invalid user input is always caught.

Reply:
I've changed the assert to raising an assertionerror and also added another line to testing (to make sure its raised).